src/routing/RouteSelector.jsx

```diff
@@ -146,9 +138,7 @@ export const RouteSelector = () => {
             "/sites/:siteName/resourceRoom",
           ]}
         >
-          <ApprovedReviewRedirect>
-            <ResourceRoom />
-          </ApprovedReviewRedirect>
+          <ResourceRoom />
         </ProtectedRouteWithProps>
 
         <ProtectedRouteWithProps path="/sites/:siteName/settings">
```

Comment:
Is there a reason why Settings is left in the existing format?

Reply:
Fixed in 5280828! The greyscale part is scoped down to just the page itself, so the sidebar remains clickable. The `<ApprovedReviewRedirect>` functionality has now been simplified to only do redirecting.